.github/workflows/want-lgtm-all.yml

```diff
@@ -0,0 +1,165 @@
+name: 'want-lgtm-all'
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+      - ready_for_review
+      - review_requested
+      - review_request_removed
+  pull_request_review:
+    types:
+      - submitted
+      - dismissed
+      - edited
+  workflow_call:
+  workflow_dispatch:
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+permissions:
+  actions: 'write'
+  pull-requests: 'read'
+
+jobs:
+  validate:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - id: 'validate-event'
+        name: 'Validate GitHub Event'
+        uses: 'actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410' # ratchet:actions/github-script@v6
+        with:
+          script: |-
+            // validate required event names
+            if (!["pull_request", "pull_request_review"].includes(context.eventName)) {
+              core.setFailed(
+                "want-lgtm-all only supports the pull_request and pull_request_review event triggers. Please modify your workflow and try again."
+              );
+              return;
+            }
+
+      - id: 'want-lgtm'
+        name: 'Validate PR Body'
+        uses: 'actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410' # ratchet:actions/github-script@v6
+        with:
+          result-encoding: 'string'
+          script: |-
+            const body = context.payload.pull_request.body || "";
+
+            // if PR body does not contain want_lgtm_all, exit gracefully
+            if (!body.match(/want_lgtm_all/gim)) {
```

Comment:
This seems far less efficient than `body.includes('want_lgtm_all')`? Even `body.toLowerCase().includes('want_lgtm_all')` would probably be more efficient.